transformation/plugins/org.jboss.tools.fuse.transformation.core/src/org/jboss/tools/fuse/transformation/core/dozer/DozerMapperConfiguration.java

```diff
@@ -50,7 +50,7 @@
     private static final String CUSTOM_MAPPER_ID = "_customMapping"; //$NON-NLS-1$
     private static final String EXPRESSION_MAPPER_ID = "_expressionMapping"; //$NON-NLS-1$
     private static final String DOZER_SCHEMA_LOC =
-            "http://dozer.sourceforge.net http://dozer.sourceforge.net/schema/beanmapping.xsd"; //$NON-NLS-1$
+            "http://dozermapper.github.io/schema/bean-mapping http://dozermapper.github.io/schema/bean-mapping.xsd"; //$NON-NLS-1$
```

Comment:
the exact url is https://dozermapper.github.io/schema/bean-mapping.xsd
Eclipse is not able to follow redirection and display an error